apps/web/components/ui/NoCalendarConnectedAlert/__tests__/NoCalendarConnectedAlert.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { cleanup, screen, render } from "@testing-library/react";
+
+import { trpc } from "@calcom/trpc/react";
+
+import NoCalendarConnectedAlert from "..";
+
+describe("Testing NoCalendarConnectedAlert", () => {
+  describe("Render test", () => {
+    it("should render without crashing", () => {
+      // Disabled as its asking for full trpc useQuery response
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      jest.spyOn(trpc, "useQuery").mockReturnValue({
+        isSuccess: true,
+        isFetched: true,
+        data: { connectedCalendars: [], destinationCalendar: undefined },
+      });
```

Comment:
With this we can mock our desired trpc useQuery responses and components should comply.